src/goto-symex/symex_goto.cpp

```diff
@@ -406,15 +406,15 @@ void goto_symext::loop_bound_exceeded(const expr2tc &guard)
   if (partial_loops && !config.options.get_bool_option("termination"))
     return;
 
-  const irep_idt &loop_id = cur_state->source.pc->location.loopid();
+  unsigned loop_id = cur_state->source.pc->loop_number;
```

Comment:
Let's rename this variable to `loop_number` just to avoid confusion.

Reply:
I agree with @fbrausse.